### DISCUSSION TITLE
JWT Authentication

### DISCUSSION THREAD

**Discussion by thienvc** [dnnsoftware/Dnn.Platform#4263] [General] [answered]
Is jwt authentication vulnerable to a brute force attack? Because there is not a similar captcha like webform login. Is it necessary to equip IPCount as shown in DnnImageHandler?

**Comment by mitchelsellers** [comment, marked as answer]
All security related questions should be addressed to the proper security location. (security@dnnsoftware.com)

However, in this situation, the underlying API's provide the protection for brute force attacks and this is not a risk.

**Reply by thienvc** [reply]
Oh, so that's it. Thank you.
